src/main/java/com/project/stockexchangeappbackend/dto/StockAmountUpdateDTO.java

```diff
@@ -0,0 +1,26 @@
+package com.project.stockexchangeappbackend.dto;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+
+@NoArgsConstructor
+@Builder
+@AllArgsConstructor
+@Data
+@ApiModel(description = "Object used to get information about updating stock amount")
+public class StockAmountUpdateDTO {
+
+    @NotNull(message = "This field is required")
+    @ApiModelProperty(notes = "User's id")
+    Long userId;
```

Comment:
It had been much better if you would use OwnerDTO here. There should be capability to change amount of stock for given amount  and give new stocks to many users (if admin is adding stocks) or take stocks away from many users (when admin is taking away stocks). Now we can just add  new stocks/or take away stock from one user